app/src/main/java/com/amaze/filemanager/ui/dialogs/ColorPickerDialog.java

```diff
@@ -22,41 +22,42 @@
 
 import com.afollestad.materialdialogs.Theme;
 import com.amaze.filemanager.R;
-import com.amaze.filemanager.ui.colors.ColorPreferenceHelper;
+import com.amaze.filemanager.application.AppConfig;
 import com.amaze.filemanager.ui.colors.UserColorPreferences;
 import com.amaze.filemanager.ui.fragments.preference_fragments.PreferencesConstants;
 import com.amaze.filemanager.ui.theme.AppTheme;
 import com.amaze.filemanager.ui.views.CircularColorsView;
-import com.amaze.filemanager.ui.views.preference.SelectedColorsPreference;
 import com.amaze.filemanager.utils.Utils;
 
+import android.app.Dialog;
 import android.content.Context;
 import android.content.SharedPreferences;
 import android.content.res.ColorStateList;
 import android.content.res.Resources;
-import android.content.res.TypedArray;
 import android.graphics.Color;
 import android.os.Build;
 import android.os.Bundle;
 import android.os.Parcel;
 import android.os.Parcelable;
-import android.util.AttributeSet;
 import android.view.LayoutInflater;
 import android.view.View;
-import android.view.Window;
 import android.widget.LinearLayout;
 import android.widget.RadioButton;
 import android.widget.TextView;
 
+import androidx.annotation.NonNull;
 import androidx.core.util.Pair;
+import androidx.preference.Preference.BaseSavedState;
+import androidx.preference.PreferenceDialogFragmentCompat;
+import androidx.preference.PreferenceManager;
 
 /**
  * This is only the dialog, that shows a list of color combinations and a customization and random
  * one.
  *
  * @author Emmanuel on 11/10/2017, at 12:48.
  */
-public class ColorPickerDialog extends SelectedColorsPreference {
```

Comment:
Why did the superclass change?

Reply:
AndroidX preference's structure of `DialogPreference` (model) and `DialogPreferenceFragment` (view) separation.

[Reference](https://stackoverflow.com/a/59736110)